samples/apps/copilot-chat-app/README.md

```diff
@@ -0,0 +1,78 @@
+3.	The sample uses two applications, a front-end web UI, and a back-end API server.  First, let’s set up and verify the back-end API server is running.
+
+    1. Navigate to `\samples\apps\copilot-chat-app\SKWebApi`
+    2.	Update `appsettings.json` with these settings:
+
+          *	Under the `“CompletionConfig”` block, make the following configuration changes to match your instance:
+
+            * `“AIService”: “AzureOpenAI”`, or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-davinci-003”,` or whichever option is appropriate for your instance.  
+            * `“Endpoint”:` “Your Azure Endpoint address, i.e. http://contoso.openai.azure.com”  If you are using OpenAI, leave this blank.
+            * You will insert your Azure endpoint key during build of the backend API Server
+
+        * Under the `“EmbeddingConfig”` block, make sure the following configuration changes to match your instance:
+            * `“AIService”: “AzureOpenAI”,` or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-embedding-ada-002”,` or whichever option is appropriate for your instance.    
+            *	You will insert your Azure endpoint key during build of the backend API Server
+            
+4. Build the back-end API server by following these instructions:
+    1.	In the terminal navigate to  `\samples\apps\copilot-chat-app\SKWebApi`
+    2. Run the command: `dotnet user-secrets set "CompletionConfig:Key"  "YOUR OPENAI KEY or AZURE OPENAI KEY"`
+    3. Run the command: `dotnet user-secrets set "EmbeddingConfig:Key" "YOUR OPENAI KEY or AZURE OPENAI KEY"`
+    4. Execute the command `dotnet build`
+    5.	Once the build is complete, Execute the command `dotnet run`
+    6.	Test the back-end server to confirm it is running.
+        * Open a web browser, and navigate to `https://localhost:40443/probe`
+        * You should see a confirmation message: `Semantic Kernel service is up and running`
+>Note: you may need to accept the locally signed certificate on your machine in order to see this message.  It is important to do this, as your browser may need to accept the certificate before allowing the WebApp to communicate with the backend.
+>Note: You may need to acknowledge the Windows Defender Firewall, and allow the app to communicate over private or public netowrks as appropriate.
+5.	Now that the back-end API server is setup, and confirmed operating, let’s proceed with setting up the front-end WebApp.
+    1. Navigate to `\apps\copilot-chat-app\webapp`
+    2.	Copy `.env.example` into a new file with the name “`.env`” and make the following configuration changes to match your instance:
+    3. Use the Application (client) ID from the Azure Portal steps above and paste the GUID into the .env file next to `REACT_APP_GRAPH_CLIENT_ID= `
+    4.	Execute the command `Yarn Install`
```

Comment:
Nit:  these two commands should be all in lower case.  On Windows, it doesn't matter much, but it does on Mac and Linux

Reply:
nice catch!